test/acceptance/interfaces/tdd.js

```diff
@@ -38,4 +38,24 @@ suite('Array', function(){
       done();
     });
   });
+
+  suite('tear down should be executed', function () {
+    var shouldBeChangeInTearDown;
+
+    setup(function () {
+      shouldBeChangeInTearDown = 5;
+    });
+
+    test('spec2', function () {
+      this.skip();
+    });
+
+    teardown(function () {
+      shouldBeChangeInTearDown = 10;
+    });
+
+    suiteTeardown(function () {
+      shouldBeChangeInTearDown.should.eql(10);
+    });
+  });
 });
```

Comment:
I'd suggest just a boolean flag (and without duplicating "should" in its name), e.g.:
```js
var tornDown;

setup(function () {
  tornDown = false;
});

// ...

tearDown(function () {
  tornDown = true;
});

suiteTeardown(function () {
  tornDown.should.be.ok();
});
```